<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialWin</class>
 <widget class="QMainWindow" name="SerialWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>32</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="openLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../icons/open.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="closeLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../icons/close.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="clearLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../icons/clear.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../icons/about.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>2</y>
       <width>151</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="infoButton">
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QComboBox" name="dataCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>105</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="parityCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>139</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="stopCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>173</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="flowCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>206</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>105</y>
       <width>57</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>DataBit:</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>139</y>
       <width>45</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Parity:</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>206</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>FlowControl:</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>173</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>StopBit:</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QComboBox" name="baudCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>71</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="nameCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>37</y>
       <width>87</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>71</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>BaudRate:</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>37</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>PortName:</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>240</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="refLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/refresh.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>441</width>
      <height>243</height>
     </rect>
    </property>
    <property name="title">
     <string>SendBox</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="sendtext">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>441</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>RecieveBox</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="recievetext">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
